ros_buildfarm/templates/release/binarydeb_job.xml.em

```diff
@@ -102,7 +102,6 @@ but disabled since the package is blacklisted (or not whitelisted) in the config
         ' ' + os_code_name +
         ' ' + arch +
         ' ' + ' '.join(repository_args) +
-        ' --binarydeb-dir $WORKSPACE/binarydeb' +
         ' --dockerfile-dir $WORKSPACE/docker_generating_docker' +
```

Comment:
The same might be the case for this option.

Reply:
Looks like this option specifies the output location for this script, so it should stay.

Reply:
Oof, actually, its more complicated than that. It's both. I'll see if there is a good way to handle this one.